src/lib/shared/components/ui/FormElements/FormRenderer/FormRenderer.stories.tsx

```diff
@@ -150,6 +152,7 @@ export const PrefilledForm: StoryFn = () => {
                     confirmPassword: 'password',
                     description: 'I am a not a robot',
                     state: 'California',
+                    radio: 'One',
```

Comment:
```suggestion
                    favoriteNumber: 'One',
```

Maybe we should have the key be more descriptive of the value it's capturing, not the element it represent.